src/views/index.html

```diff
@@ -22,14 +22,15 @@
     <nav class="navbar navbar-inverse">
       <div class="container-fluid">
         <div class="navbar-header">
-          <a class="navbar-brand" href="#">
+          <a class="navbar-brand" href="#" onClick="window.location.reload()">
             <img src="../static/images/logo-acc.png" style="width: 50%" />
           </a>
         </div>
         <ul class="nav navbar-nav navbar-right">
           <li><a href="#">Home</a></li>
           <li><a href="#how-to-join">Join</a></li>
           <li><a href="#projects">Projects</a></li>
+          <!-- <li><a href="#officer-team">Officers</a></li> -->
```

Comment:
Can u just delete this instead? It keeps the code base clean. You can always add it back later.